golang/simple_consumer.go

```diff
@@ -142,7 +146,11 @@ func (sc *defaultSimpleConsumer) Subscribe(topic string, filterExpression *Filte
 }
 
 func (sc *defaultSimpleConsumer) Unsubscribe(topic string) error {
-	sc.cli.getMessageQueues(context.Background(), topic)
```

Comment:
Getting message queues is redundant for topic unsubscription?

Reply:
it makes sense, i'll update it with this pr

Reply:
> Getting message queues is redundant for topic unsubscription?

Done